demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -776,3 +778,39 @@ def test_validate_author_image_exists_invalid(self, repo):
             assert not res
             assert f'Partners must provide a non-empty author image under the path {author_image_path}.' in \
                    self.validator.get_errors()
+
+    @pytest.mark.parametrize('pack_metadata, rn_version, create_rn, expected_results', [
+        ({"currentVersion": "1.0.1"}, "1.0.1", True, True),
+        ({"currentVersion": "1.0.2"}, "1.0.1", True, False),
+        ({"currentVersion": "1.0.1"}, "1.0.2", True, False),
+        ({"currentVersion": "1.0.1"}, "1.0.2", False, False),
+        ({"currentVersion": "1.0.0"}, "1.0.2", False, True),
```

Comment:
Since *create RN* is False, no RN should be created, therefore **_get_pack_latest_rn_version** should return an empty String and **_is_right_version** should enter the condition  if not *rn_version* and *current_version* == '1.0.0' which is mocking the case where a new integration is created, therefore is version is 1.0.0 and it has no RN.
Anyway, changed to 1.0.0.